modules/imap/output_modules.php

```diff
@@ -370,27 +370,31 @@ protected function output() {
             $txt .= ' | <a class="hlink" id="copy_message" href="#">'.$this->trans('Copy').'</a>';
             $txt .= ' | <a class="hlink" id="move_message" href="#">'.$this->trans('Move').'</a>';
             $txt .= ' | <a class="archive_link hlink" id="archive_message" href="#">'.$this->trans('Archive').'</a>';
-            $txt .= ' | ' . snooze_dropdown($this, isset($headers['X-Snoozed']));
+            if (!isset($headers['Flags']) || !stristr($headers['Flags'], 'draft')) {
```

Comment:
Why not create a variable named for example is draft which will hold this value

`$is_draft = isset($headers['Flags']) && stristr($headers['Flags'], 'draft');`

Then test with is draft in multiple if that are below